legal-api/migrations/versions/33501a263f32_sp_gp.py

```diff
@@ -18,11 +18,9 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column('businesses', sa.Column('business_start_date', sa.DateTime(timezone=True), nullable=True))
```

Comment:
Will remove from dev manually (removing from migration script since it is not moved to test or prod)